piControlMain.c

```diff
@@ -385,18 +393,12 @@ static int piControlReset(tpiControlInst * priv)
 		// ignore errors
 	}
 
-	if (piDev_g.machine_type == REVPI_CORE) {
-		PiBridgeMaster_Reset();
-	} else if (piDev_g.machine_type == REVPI_CORE_SE) {
-		PiBridgeMaster_Reset();
-	} else if (piDev_g.machine_type == REVPI_CONNECT) {
-		PiBridgeMaster_Reset();
-	} else if (piDev_g.machine_type == REVPI_CONNECT_SE) {
-		PiBridgeMaster_Reset();
-	} else if (piDev_g.machine_type == REVPI_COMPACT) {
+	if (piDev_g.machine_type == REVPI_COMPACT) {
 		revpi_compact_reset();
 	} else if (piDev_g.machine_type == REVPI_FLAT) {
 		revpi_flat_reset();
+	} else if (piDev_g.pibridge_supported) {
+		PiBridgeMaster_Reset();
```

Comment:
Not sure why the PiBridge check is now at the end of the if-else ladder instead of at the beginning where it was previously. (No biggie of course.)